### ISSUE TITLE
Separate service taxon audit into areas for Veteran research

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#32055]
## Issue Description
Leyda's audit from #30839 provided a comprehensive list of areas for future research. Before we can process with the first round of research, we need to identify/prioritize areas for focus

---
## Tasks

Review the new services tab from [audit-spreadsheet-vha-health-services-Oct2021.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7421997/audit-spreadsheet-vha-health-services-Oct2021.xlsx) and 
- [x] Identify those services which are clearly benefit services only (not provided on a VA Medical center campus) and move them into an issue for future benefit taxonomy research
- [x] Use our best guesses to map remaining items to the  VHA services or top tasks available on VA medical center pages, and their category and add to a list to be to _validated_ with Veteran research (ie: Refill my prescription = Pharmacy, categorized as  Primary Care)
- [x] Identify tasks/services which do not easily map to existing taxon and add to a list to be _discovered_ with Veteran research (ie: Medical records, Chaplain)

Review the Audit #30839 tab from [audit-spreadsheet-vha-health-services-Oct2021.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/7421997/audit-spreadsheet-vha-health-services-Oct2021.xlsx) and 
- [x] Identify the services which have been flagged for research due to name or misalignment of category and add to a list to be to _validated_ with Veteran research (ie: Women Veteran care, categorized as Specialty care)
- [x] Identify the services which are categorized as "Other" and add to a list to be _discovered_ with Veteran research 

## Acceptance Criteria
- New issue containing list of benefit services for future Veteran research
- List of tasks/services with their matching service taxons and current categorization to be validated with Veteran research for use in #31626 
- New issue for a discovery research plan containing the list of tasks/services with unmatched taxons and/or categories

**Comment by nicko-amida** [comment]
- This list refers to the "Research Groups" tab of the [services-research-categorization-Oct2021.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/medical-centers/services-taxonomy/taxonomy-research-2021/services-research-categorization-Oct2021.xlsx) spreadsheet.
- There are a large number of potential services/categories, so they may not all be included in the same study. The research plan for [#31626](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31626) will be more specific about which services/categories to focus on.

Here's a breakdown of what's in the speadsheet:

- **"New" services - category research**
     - **Validating catetory**: identifies new services that seem to map to existing categories, which research will validate/invalidate
     - **Discovering category**: identifies new services that don't seem to map to existing  categories. Research will help determine if Veterans prefer existing categories, or creating new categories, for these services.

- **"New" services - service/task research**
     - **Validating service/task**: identifies new services that seem to map to existing VHA services, or top tasks on VAMC pages. Research will validate/invalidate.
     - **Discovering service/task**: identifies new services to don't seem to map to existing VHA services, or VAMC top tasks. Research will help determine which services and tasks apply, or if new services/tasks are appropriate.
